### PR TITLE
Cleanup strongswan table 220 when master gw becomes non-master.

### PR DESCRIPTION
**Pull request by mangelajo**
HA Failover tests start working flawlesly after this change, otherwise
sometimes the node that previously was a master gw, left over the 220
routing table, causing conflict with the default routes managed by
the route agent.

Fixes: #346

### CODE REVIEW

File: pkg/routeagent/controllers/route/route.go [hunk 1]
```diff
@@ -597,6 +601,15 @@ func (r *Controller) cleanRoutes() {
 	}
 }
 
+func (r *Controller) cleanStrongswanRoutingTable() {
+	cmd := exec.Command("/sbin/ip", "r", "flush", "table", "220")
```
Comment (sridhargaddam) [inline pkg/routeagent/controllers/route/route.go]: This API is tied to Strongswan implementation and now that there are some efforts in adding other drivers, shall we add a comment to revisit this API?

Comment (mangelajo) [reply pkg/routeagent/controllers/route/route.go]: True, let me extend L467 note to the functions.

I believe this should not be altogether here. Also, in cases where the route agent won't live on every node this won't work.

We need to find a better place, the alternative would be having a separate daemonset to run on nodes, but that sounds expensive... we need to keep thinking about it.

Comment (mangelajo) [reply pkg/routeagent/controllers/route/route.go]: I updated the comments, let me know if it looks ok :)

Comment (sridhargaddam) [reply pkg/routeagent/controllers/route/route.go]: Thanks for adding the comments.

**Pull request merged**